core/src/ons/ons.js

```diff
@@ -55,7 +55,9 @@ ons._animationOptionsParser = animationOptionsParser;
 ons._autoStyle = autoStyle;
 ons._DoorLock = DoorLock;
 ons._contentReady = contentReady;
-ons.defaultPageLoader = defaultPageLoader;
+ons.defaultPageLoader = (pageLoader)=>{
+   return defaultPageLoader=pageLoader;
```

Comment:
@badlee it's a nice idea but shouldn't the default one be kept if the user doesn't provide a custom one?

Like `return pageLoader ? defaultPageLoader = pageLoader : defaultPageLoader;`